doc/content/getting-started/migrating/device-json.md

```diff
@@ -45,6 +45,26 @@ Multiple end devices can also be contained in a single `devices.json` file like
 ]
 ```
 
+***Important observation***
+
+The format described above works for the console, when using the cli a JSON stream is expected. One of the possible ways of doing it in the cli would be:
```

Comment:
```suggestion
The format described above works for the Console. However, when using the CLI a JSON stream is expected. You can pipe multiple files:
```

Reply:
Section removed in favour of having a more simple explanation. I updated the object reference to remove `[` and `]`, since the json stream works both on Console and CLI there is no need to have different file formats for each of them. 

Another reason to remove the section was the comment made by Nejraselimovic where was pointed where the usage of the file is already described in another page.